docs/testing.md

```diff
@@ -0,0 +1,148 @@
+# Testing Using Docker
+
+**Table of Contents**
+
+- [Installation](#Installation)
+  - [Docker configuration](#docker-configuration)
+  - [Python configuration](#python-configuration)
+- [Basic Docker commands](#basic-docker-commands)    
+
+
+## Installation
+
+#### Docker configuration
+
+The base Docker installation methods for popular Linux distributions are listed here:
+	https://docs.docker.com/engine/install/
+
+Docker is not supported by **Fedora 33** officially. On my host system I used the following method (Moby based approach):
+
+1. Delete the old version of docker (if it was installed)
```

Comment:
Can we fold that using e.g. collapsive section (like described [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab))

Reply:
Done